rep-2000.rst

```diff
@@ -306,6 +306,132 @@ Build System Support:
 - cmake
 - setuptools
 
+Dashing Diademata (May 2019 - May 2021)
+---------------------------------------------------------------
+
+Support Tiers:
+
+Tier 1
+
+Tier 1 platforms are subjected to our unit test suite and other testing tools on a frequent basis including continuous integration jobs, nightly jobs, packaging jobs, and performance testing.
+Errors or bugs discovered in these platforms are prioritized for correction by the development team.
+Significant errors discovered in Tier 1 platforms can impact release dates and we strive to resolve all known high priority errors in Tier 1 platforms prior to new version releases.
+
+Tier 2
+
+Tier 2 platforms are subject to ad hoc testing and binary packages may not be provided or supported.
+Tier 2 platforms may have downloadable binary archives.
+Errors may be present in released product versions for Tier 2 platforms.
+Known errors in Tier 2 platforms will be addressed subject to resource availability on a best effort basis and may or may not be corrected prior to new version releases.
+
+Tier 3
+
+Tier 3 platforms are those for which community reports indicate that the release is functional.
+The development team does not run the unit test suite or perform any other tests on platforms in Tier 3.
+Installation instructions should be available and up-to-date in order for a platform to be listed in this category.
+Community members may provide assistance with these platforms.
```

Comment:
We should move the tier descriptions somewhere above the distros, because the description is common to all of them.

Reply:
The tier language was only introduced with Crystal and is still getting revised. I don't mind moving it to the top if we want to standardize them.

Reply:
I believe the idea is that the tier definitions are the same across all versions. CC @tfoote

Reply:
Using @tfoote's recently updated Crystal tier descriptions I moved them to the top of the REP.